resources/views/review.blade.php

```diff
@@ -1,3 +1,38 @@
+@php
+    $formattedDate = formatDate($review->created_at);
+    $backLabel = request('from') === 'movie' ? 'Back to movie' : (request('from') === 'profile' ? 'Back to profile' : 'Back');
+    $backHref = url()->previous();
+@endphp
+
 <x-layout>
-    <h1>Review</h1>
+    <x-section-header.back-link
+        title="{{ $review->movie->title }}"
```

Comment:
```suggestion
        title="{!! $review->movie->title !!}"
```

to not get the unicode version of special characters like in your screenshot